lib/Varien/Image/Adapter/Gd2.php

```diff
@@ -182,7 +182,7 @@ public function save($destination = null, $newName = null)
         $functionParameters[] = $this->_imageHandler;
         $functionParameters[] = $fileName;
 
-        // set quality param for JPG file type
+        $quality = $this->quality() ?? Mage::getStoreConfigAsInt('system/media_storage_configuration/image_quality');
```

Comment:
This var doesn't seem to be used, unless you meant to replace `$this->getQuality()` in the below lines?

Reply:
😂 you're right